### ISSUE TITLE
Show initial value for scalar markets

### ISSUE THREAD

**Issue by maltekliemann** [zeitgeistpm/ui#596]
While deploying a pool for a scalar market and setting the initial odds, it would be very useful to show what the initial predicted value will be.

For example, suppose I'm creating a market with values between 295 and 300 and I want the initial prediction to be 297. Then I need to manually calculate (297 - 295) / (300 - 295) = 0.4 to figure out what the initial value of ~~SHORT~~ LONG should be. Would be more straightforward if I could just use a slider/textbox to set the initial value and the values of LONG and SHORT are calculated accordingly.

To calculate the value of ~~SHORT~~ LONG from an initial prediction x of [a, b], you just do (x - a) / (b - a).

**Comment by Robiquet** [comment]
@maltekliemann is this resolved by this? 
<img width="841" alt="image" src="https://user-images.githubusercontent.com/4950844/233026866-2df39fad-0248-460c-84a1-b2b8c1840636.png">

**Comment by maltekliemann** [comment]
No, unfortunately. I mean, depends on where that bar is displayed. What I'm proposing is that when deploying a pool, I should be able to see which prediction my initial prices correspond to, or, even better, be able to set the initial prediction and let the UI calculate the initial prices.

For example, if I want to create a market about the number of tornados in the U.S. and my range is 400-800, then I want to be able to specify that the initial prediction should be 555 and have the prices set accordingly.
